cg/cli/analysis.py

```diff
@@ -12,14 +12,16 @@
 LOG = logging.getLogger(__name__)
 PRIORITY_OPTION = click.option('-p', '--priority', type=click.Choice(['low', 'normal', 'high']))
 EMAIL_OPTION = click.option('-e', '--email', help='email to send errors to')
+START_WITH_PROGRAM = click.option('-sw', '--startwith', help='start mip-pipeline with this program and run downstream processes')
 
 
 @click.group(invoke_without_command=True)
 @PRIORITY_OPTION
 @EMAIL_OPTION
+@START_WITH_PROGRAM
 @click.option('-f', '--family', 'family_id', help='family to prepare and start an analysis for')
 @click.pass_context
-def analysis(context, priority, email, family_id):
+def analysis(context, priority, email, family_id, startwith):
```

Comment:
... like here: `startwith` in python (as opposed to in the CLI), will be named `start_with`